File: app/src/main/java/com/beone/bestpractice/adapter/HeaderViewHolder.java
package com.beone.bestpractice.adapter;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.beone.bestpractice.R;

public class HeaderViewHolder extends RecyclerView.ViewHolder{
  private TextView textView;

  public HeaderViewHolder(@NonNull View itemView) {
    super(itemView);
    textView = itemView.findViewById(R.id.tvTitle);
  }

  public void onbind(String data) {
      textView.setText(data);
  }
}

File: app/src/main/java/com/beone/bestpractice/core/ui/HeaderViewHolder.kt
package com.beone.bestpractice.core.ui.adapter

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.beone.bestpractice.R

class HeaderViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    private val textView: TextView
    fun onbind(data: String?) {
        textView.text = data
    }

    init {
        textView = itemView.findViewById(R.id.tvTitle)
    }
}